<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ui_MoveRotateOptions</class>
 <widget class="QWidget" name="Ui_MoveRotateOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>24</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>128</width>
    <height>22</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Move Rotate Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item row="0" column="10">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="11">
    <widget class="QCheckBox" name="cbCurrentAttr">
     <property name="toolTip">
      <string>If checked, current attributes will be applied to created entities, otherwise original ones will be used. </string>
     </property>
     <property name="text">
      <string>Use Current Attributes</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QCheckBox" name="cbMultipleCopies">
     <property name="toolTip">
      <string>If checked, multiple copies will be created. </string>
     </property>
     <property name="text">
      <string>Multiple Copies</string>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QCheckBox" name="cbSameAngleForCopies">
     <property name="toolTip">
      <string>If checked, all copies will be rotated by the same agle. Otherwise, angle will be proportional to number of each copy. </string>
     </property>
     <property name="text">
      <string>Same angle for copies</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="leAngle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Rotation angle. </string>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QSpinBox" name="sbNumberOfCopies">
     <property name="toolTip">
      <string>Number of copies.</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QCheckBox" name="cbFreeAngle">
     <property name="toolTip">
      <string>If checked, rotation angle is defined by mouse. Otherwise, specified value is used.</string>
     </property>
     <property name="text">
      <string>Free</string>
     </property>
    </widget>
   </item>
   <item row="0" column="13">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Angle:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="12">
    <widget class="QCheckBox" name="cbCurrentLayer">
     <property name="toolTip">
      <string>If checked, created entities will be placed into current layer, otherwise the will be in original layers. </string>
     </property>
     <property name="text">
      <string>Use Current Layer</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QCheckBox" name="cbKeepOriginals">
     <property name="toolTip">
      <string>If checked, original entities will survive, otherwise they will be removed. </string>
     </property>
     <property name="text">
      <string>Keep Originals</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
